delivery_location/migrations/0002_test_data.py

```diff
@@ -7,25 +7,20 @@
 class Migration(migrations.Migration):
     dependencies = [
         ('delivery_location', '0001_initial'),
+        ('supplier', '0002_test_data'),
     ]
 
     def generate_data(apps, schema_editor):
         with transaction.atomic():
             test_data = [
-                ('worldMaster', 'ash', 'katchamp', 'pokemon123', 'pokemon inc'),
-                ('ed', 'edd', 'eddy', 'jawbreaker101', 'jawbreaker inc'),
+                ('worldMaster'),
+                ('ededed'),
             ]
 
-            for userName, firstName, lastName, passwrd, businessName in test_data:
-                sup = Supplier(
-                    user_name=userName,
```

Comment:
so we don't need to create this Supplier anymore? do we still need all this test_data?

Reply:
the suppliers are taken from supplier 0002_test_data

Reply:
Test data i have updated it to match only the usernames of the suppliers because all the rest is useless and we dont need to create a new supplier we need to match it with existed supplier